package.json

```diff
@@ -112,7 +113,7 @@
     "remark-highlight.js": "^6.0.0",
     "remark-html": "^13.0.1",
     "remark-parse": "^9.0.0",
-    "remixicon": "^2.5.0",
+    "remixicon": "3.3.0",
```

Comment:
nitpick: not sure if it's voluntary so I'm asking, do we want to be locked on the `3.3.0` or do we go for the same as before `^3.3.0`
I don't know much about the Remix project but if they are serious with semver and breaking changes, we could keep the `^`.

Reply:
This is voluntary: Since the operation (transforming remix icons into .js) is now totally manual, there is no purpose in using the `^` notation.

Reply:
Got it, thanks for the answer, this is perfect, LGTM then ! :smile: